pkg/apis/certificates/v1beta1/defaults_test.go

```diff
@@ -69,6 +82,17 @@ func TestIsKubeletServingCSR(t *testing.T) {
 			usages: kubeletServerUsages[1:],
 			exp:    false,
 		},
+		"does not default to kubelet-serving if it is missing an expected usage if allow omitting key encipherment": {
+			req:                               newCSR(kubeletServerPEMOptions),
+			usages:                            kubeletServerUsagesNoRSA[1:],
+			allowOmittingUsageKeyEncipherment: true,
+			exp:                               false,
+		},
+		"does not default to kubelet-serving if it is missing an expected usage withou key encipherment": {
+			req:    newCSR(kubeletServerPEMOptions),
+			usages: kubeletServerUsagesNoRSA[1:],
```

Comment:
should test kubeletServerUsagesNoRSA without allowOmittingUsageKeyEncipherment, right?

```suggestion
			usages: kubeletServerUsagesNoRSA,
```

Reply:
Yes. It's a copy-and-paste mistake.